docs/process/process_areas/verification/guidance/verification_process_toolreqs.rst

```diff
@@ -0,0 +1,117 @@
+..
+   # *******************************************************************************
+   # Copyright (c) 2025 Contributors to the Eclipse Foundation
+   #
+   # See the NOTICE file(s) distributed with this work for additional
+   # information regarding copyright ownership.
+   #
+   # This program and the accompanying materials are made available under the
+   # terms of the Apache License Version 2.0 which is available at
+   # https://www.apache.org/licenses/LICENSE-2.0
+   #
+   # SPDX-License-Identifier: Apache-2.0
+   # *******************************************************************************
+
+Test Linking to Requirements
+============================
+
+
+.. gd_req:: Linking Requirements to Tests
+    :id: gd_req__link_tests
+    :status: valid
+    :complies: std_req__iso26262__support_8
+
+    For linking test suites to requirements following metadata shall be used:
+
+    * Verifies
+    * Description
+    * Status
+        * valid
+        * invalid
+    * TestType
+        * Requirements-based test (requirements-based)
+        * Design based tests (design-based)
+        * Interface Test (interface-test)
+    * DerivationTechnique
+        * Analysis of requirements (requirements-analysis)
+        * Analysis of design (design-analysis)
+        * Analysis of boundary values (boundary-values)
+        * Analysis of equivalence classes (equivalence-classes)
+        * Error guessing based on knowledge or experience (error-guessing)
+
+.. gd_req:: Linking Requirements to Tests (C++)
+    :id: gd_req__link_tests_cpp
+    :status: valid
+    :complies: std_req__iso26262__support_8
+
+    For linking C++ test suites to requirements **record properties** shall be used. Attributes which are common for all test cases can be specified in the Setup Function (SetUp()), the other attributes which are specific for each test case need to be specified within the test case:
+
+    .. code-block:: cpp
+
+      class TestSuite : public ::testing::Test{
+         public:
+            void SetUp() override
+            {
+               RecordProperty("Status", "<Status>");
+               RecordProperty("ASIL", "<ASIL>");
+               RecordProperty("Priority", "<Priority>");
```

Comment:
@hoe-jo  Do we really want to have Priority a property? Also it is not described anywhere so far and we want to execute all tests in CI. I wonder, if we really need it.